src/main/java/ipman/Parser.java

```diff
@@ -0,0 +1,160 @@
+package ipman;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * Parser class
+ * @author miloaisdino
+ */
+public class Parser {
+    private final ArrayList<Task> list;
```

Comment:
Since this is an ArrayList of Tasks, perhaps you could have named it in plural form, such as `tasks` instead to clearly distinguish that this variable is a collection of multiple items?